### ISSUE TITLE
[Bug] Linux Screen commnad output format error

### ISSUE THREAD

**Issue by sti1l-0** [kingToolbox/WindTerm#2187]
when using editor like vim or other command that fresh the terminal on the same place, windterm seems cannot duel with "clear" correctly. Like below I just page down in vim, the bottom bar has take all my screen. (you could see the “~@k”, some other command output others)
![image](https://github.com/kingToolbox/WindTerm/assets/37724877/b395be7f-ca1a-43a4-9b4e-57b2d2d84ca5)

**Comment by iaasweb** [comment]
yep, I'm facing the same error like this, and someone already post this issue https://github.com/kingToolbox/WindTerm/issues/2171

**Comment by sti1l-0** [comment]
> yep, I'm facing the same error like this, and someone already post this issue #2171

dose not help

**Comment by czxin788** [comment]
I will use elcterm instead of windterm

**Comment by Ansen** [comment]
> I also encountered the same issue. Referring to the solution in 2171, I changed the `term` option in the "session property" to `xterm`, and it seems the issue no longer occurs (in ubuntu 22.04). In centos, it seems still occurs.

same issue in 2.7.0 pre-release 3
